src/components/SUSHIControls.js

```diff
@@ -215,7 +215,7 @@ export default function SUSHIControls(props) {
               margin="dense"
               fullWidth
               label="Dependencies"
-              helperText="dependencyID#version, dependencyID#version"
+              helperText="Format: packageId#version, packageId#version (ex. hl7.fhir.us.core#3.1.1)"
```

Comment:
This is a nit, but isn't it customary to use `e.g.,` in a situation like this rather than `ex.`?

Reply:
I almost used that, so I'm happy to swap it out!

Reply:
21b4522 swapped!